.github/workflows/coding-conventions.yml

```diff
@@ -35,8 +35,8 @@ jobs:
       - name: Run eslint
```

Comment:
It looks to me like you are adopting the brand casing for GitHub, Bun, etc. in this PR. We should probably do the same for __ESLint__.